### PR TITLE
Add sql command to update user profile

### LINKED ISSUE #58: Add sql command to update user profile
### Add sql command to update user profile

Base: `main`
New branch: `sql-58-update-profile`

---

**Prerequisites**:

- none

---

**Requirements**:

- Filename: `update_profile.sql`
- Query must update the following fields of the user that matches the `user_id`:
  - username
  - contact
  - bio
- Make sure to also update `updated_at` to current date
- Return `user_id`

### PR DESCRIPTION
**Pull request by LyndonTomas**
Hopefully closes #58

As you can see, I have placed random variables to prove that the code does update the entry.
To conform with the syntax (the one with %1), kindly let me know where should I put each and everyone of the so I can replace the items accordingly on my next push.

Screenshots:

Before update:
before update

Update Query:

update query

After Update:
after update

### CODE REVIEW

File: src/repositories/sql/users/update_profile.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+UPDATE users
+SET
+username = $1,
+contact = $2,
+bio = $3,
+updated_at= (SELECT NOW())
+WHERE user_id=$4;
```
Comment (jp-ong) [inline src/repositories/sql/users/update_profile.sql:7]: - Add indentations for readability
```
UPDATE users
  SET
    x=y
  WHERE ...
```
- You can just write `updated_at = NOW()`
- Make spacing consistent, pick either `x = y` or `x=y`, but not both

**Review by jp-ong — CHANGES_REQUESTED** [review]

**Comment by LyndonTomas** [comment]
Here is the updated query.

Before the query:

before

Sample query:
sample query


After the query:
after

**Review by jp-ong — APPROVED** [review]
Ok

**Review by jp-ong — APPROVED** [review]
ok

**Pull request merged**